fe/fe-core/src/main/java/com/starrocks/analysis/LoadStmt.java

```diff
@@ -80,6 +80,7 @@ public class LoadStmt extends DdlStmt {
     public static final String STRICT_MODE = "strict_mode";
     public static final String TIMEZONE = "timezone";
     public static final String PARTIAL_UPDATE = "partial_update";
+    public static final String MERGE_CONDITION = "merge_condition";
```

Comment:
Merge update condition
We can support merge delete condition

Reply:
maybe just call it update_condition? it will change to thrift TExpr in the following PR.

Reply:
I think his intention/final goal is to support merge syntax? for example:
```
MERGE INTO dest
    USING src
    ON (src.id = dest.id)
  WHEN MATCHED and src.ts > dest.ts THEN
    UPDATE SET dest.address = src.address
  WHEN NOT MATCHED THEN
     delete
```

There are multiple condition exprs in merge into syntax, currently, we are not discussing it thoroughly about how to express those condition and operation collection using those load parameters.